File: build-system/tests/flavoredlib/app/src/main/java/com/android/tests/flavorlib/app/MainActivity.java
package com.android.tests.flavorlib.app;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity {
    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

        App.handleTextView(this);
        LibWrapper.handleTextView(this);
    }
}

File: build-system/integration-test/test-projects/kotlinApp/src/main/kotlin/com/example/android/kotlin/MainActivity.kt
package com.example.android.kotlin

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : Activity() {

    var text: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_layout)
        val tmpTxt = findViewById(R.id.someText)
        text = tmpTxt as TextView
        if (text != null) {
            text?.setText("testing kotlin")
            android.util.Log.d("kotlin", text?.getText().toString())
        }
        val tmpClick = findViewById(R.id.click)
        val click = tmpClick as Button
        click?.setOnClickListener { text?.setText("clicked!") }
    }
}
